scripts/spack/configs/x86_64/packages.yaml

```diff
@@ -10,11 +10,11 @@ packages:
   #    prefix: /usr/
   #  buildable: false
 
-  cmake:
```

Comment:
This is for generic non LC systems. An example of which is the github-actions docker test we run where we build and test Spheral on Ubuntu 20. This file is based off of what is available in Ubuntu 20, in this PR the minimum CMake requirement was increased to 3.18 and so we are having spack build and install a newer version of CMake to use.

Reply:
So what do we want to do with this file?  Looks like this version does work with our Docker image (I'm assuming since the CI test passed).   Should we take these changes or revert to develop for this file?

Reply:
If this works for our tests and your WSL2 instance then we should keep this in the meantime (no point holding up your dev workflow if it doesn't change anything). I'll open an issue about what we talked about today in regards to the external spack config files so that we can possibly revert them later for use with only our docker container.